cmd/utils/flags.go

```diff
@@ -761,6 +761,29 @@ var (
 		Value: metrics.DefaultConfig.InfluxDBTags,
 	}
 
+	MetricsEnableInfluxDBV2Flag = cli.BoolFlag{
+		Name:  "metrics.influxdbv2",
+		Usage: "Enable metrics export/push to an external InfluxDB v2 database",
+	}
+
+	MetricsInfluxDBTokenFlag = cli.StringFlag{
+		Name:  "metrics.influxdb.token",
+		Usage: "Token to authorize access to the database (v2 only)",
```

Comment:
This one, and the three below, all say "(v2 only)" in the Usage portion. Maybe we should just change their namespace to be `metrics.invfluxdbv2.token` etc ?

Reply:
Okay.  Also noted on the descriptions of the username/password flags that those are v1-only.